editor/editor_data.h

```diff
@@ -132,6 +132,7 @@ class EditorData {
 
 	List<PropertyData> clipboard;
 	UndoRedo undo_redo;
+	Vector<Callable> undo_redo_callbacks;
```

Comment:
Some feedback since you are going this route:

* If you are placing this in EditorData, then just expose the API to register/unregister in EditorData, which can be obtained with EditorNode::get_editor_data() isntead of EditorNode
* Since this is plugin friendly, I would expose this API (add remove undo/redo callback) in EditorPlugin.
* Name is a bit confusing, I would rename to undo_redo_inspector_hook_callback, since its basically what it is for.

Reply:
Alright, that sounds good to me. Regarding the first point I did that to mimic what what done for get_undo_redo(), but I can change that.